roles/testnode/tasks/yum/packages.yml

```diff
@@ -1,4 +1,10 @@
 ---
+# this is needed for the yum-complete-transation command next
+- name: Ensure yum_utils is present.
+  yum:
+    name: yum-utils
+    state: present
```

Comment:
I don't have a strong opinion on that, we certainly could.  Seemed to work as 'present' though.